modules/issue_tracker/php/issue_tracker_ControlPanel.class.inc

```diff
@@ -0,0 +1,184 @@
+<?php
+
+/**
+ * Imaging browser control panel class file
+ *
+ * PHP version 5
+ *
+ * @category Loris
+ * @package  Main
+ * @author   Loris team <info-loris.mni@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+*/
+
+/**
+ * Imaging browser control panel class
+ *
+ * This class provides the management GUI for the status flags
+ * and some links of the imaging browser.
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Loris team <info-loris.mni@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+class Issue_Tracker_ControlPanel
+{
+    /**
+     * Var to store the sessionID.
+     */
+    var $issueID;
+
+    /**
+     * Sets up the isssueID
+     *
+     * @param int $issueID current issueID
+     *
+     * @return NULL
+     */
+    function __construct($issueID)
+    {
+        $this->issueID = $issueID;
+    }
+
+    /**
+     * Determine whether the user has permission to view the imaging_browser page
+     *
+     * @return bool whether the user hass access
+     */
+    function _hasAccess()
+    {
+        $user =& User::singleton();
+        return $user->hasPermission('issue_tracker_can_assign');
+    }
+
+    /**
+    *  Gets data for the control panel template
+    *
+    * @return $subjectData array with data for the template
+    */
+    function getData()
+    {
+        $factory = NDB_Factory::singleton();
+        $issueData['backURL'] = $factory->settings()->getBaseURL()
+            . urldecode($_REQUEST['backURL']);
```

Comment:
No, I set "&backURL=/issue_tracker/" and nothing gets duplicated. I copied this from imaging browser.